L/LAGraph/build_tarballs.jl

```diff
@@ -0,0 +1,78 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+name = "LAGraph"
+version = v"1.0.1"
+
+sources = [
+    GitSource(
+        "https://github.com/GraphBLAS/LAGraph.git",
+        "0c84c41c561608a49a770569642eaff4bcc7eb8f",
+    ),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/LAGraph
+install_license LICENSE
+# Linux builds throw "undefined reference to 'clock_gettime' and 'clock_settime'" errors if -lrt isn't set
+if [[ "${target}" == *-linux-* ]]; then
```

Comment:
Only x86 glibc needs linking to librt, because we use a very old version there: https://github.com/JuliaPackaging/Yggdrasil/blob/a01ec85787f068f545ca66c44bce60fba9878810/RootFS.md#glibc-versions
```suggestion
if [[ "${target}" == *86*-linux-* ]]; then
```

Reply:
Oh, BTW, this is only for glibc
```suggestion
if [[ "${target}" == *86*-linux-gnu ]]; then
```